github_copilot_seats/triggers.py

```diff
@@ -0,0 +1,28 @@
+from autokitteh.slack import slack_client
+from seats import prune_idle_seats, find_idle_seats
```

Comment:
Prefer to import modules instead of classes/functions within them, unless the module name is cumbersome/redundant.

So a nice way to improve your code would be to use `import seats`, and rename the functions in it, so calls here would be `seats.prune_idle` and `seats.find_idle`.

Reply:
I'm fine with using modules instead, but I don't like the renaming of the functions. Yes, it adds brevity to `triggers.py` but it sacrifices clarity in `seats.py`.

I'd rather do `seats.find_idle_seats` etc

Reply:
importing seats created a name conflict. take a look at the solution, I think what we had before was fine considering it's a small project, but I stand by not changing the function names in `seats.py`.

Reply:
I still prefer "import seats" instead of importing the functions directly, and I think your solution (renaming the "seats" variables) is fine. If it was more complicated/convoluted than that then that would indeed be a good reason to keep the original imports.

Either way, I agree with your rationale to keep "seats" in the function names in the `seats` module.